src/lib.rs

```diff
@@ -1,23 +1,33 @@
 #![warn(clippy::all)]
-#![allow(clippy::single_match)] 
+#![allow(clippy::single_match)]
 
 mod painter;
 
 pub use painter::Painter;
 
 use {
-    clipboard::ClipboardProvider,
     egui::*,
-    sdl2::{event::WindowEvent, mouse::MouseButton, mouse::SystemCursor, keyboard::{Keycode, Mod} }
+    sdl2::{
+        event::WindowEvent,
+        keyboard::{Keycode, Mod},
+        mouse::MouseButton,
+        mouse::SystemCursor,
+    },
 };
 
-use clipboard::ClipboardContext; // TODO: remove
+#[cfg(not(feature = "clipboard"))]
+mod clipboard;
+
+use clipboard::{
```

Comment:
Does this piece of code also not require a cfg check? If we do a 'use clipboard' and that's not enabled in features, wouldn't this fail?

Reply:
I added stub clipboard implementation in the clipboard.rs, which exports similar APIs as an cli-clipboard crate, so the imports is same.